src/app/style.scss

```diff
@@ -28,6 +28,12 @@ mat-sidenav-container {
   position: absolute;
   top: variables.$toolbar-height;
   width: 100%;
+
+  // Note:
+  // WebTerminal is hidden under z-index mat-sidenav
+  & mat-sidenav.mat-drawer.mat-drawer-side {
+    z-index: 1;
```

Comment:
This style does not have any affect

Reply:
correct! This is because previously, the terminal i was implementing as `FIXED` at bottom and placing in root level that was not possible due to some UI limitation/structure + Terminal was more specific so In order to show as overlay `z-index` was causing issue to hide overlay underneath.

But now it is part of cluster module and only visible content area.

Good catch! i will remove it.

Reply:
fixed